File: C/Array/Average.c
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Accept N numbers from user and return its Average. 
//                  
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

/*
Input : N : 6 
Elements :85 66 3 80 93 88 
Output : 69.16667

*/
#include"Header.h"
int main()
{
    int iSize=0,*Arr=NULL;
    float fRet=0.0f;
    printf("Enter the Size of Array\n");
    scanf("%d",&iSize);

    Arr=(int*)malloc(iSize*sizeof(int));

    
  
   printf("Enter the elements\n");
   for(register int i=0;i<iSize;i++)
   {
       scanf("%d",&Arr[i]);
   }
   fRet=Average(Arr,iSize);
   free(Arr);

   printf("Average is:%f",fRet);

   return 0;
}

File: Java/Array/Average.java
/////////////////////////////////////////////////////////////////////////////
//
// Accept  numbers From user  Display the  Average of it.
//
////////////////////////////////////////////////////////////////////////////






import java.util.*;
import UserPackage.ArrayX;
class Logic extends ArrayX
{
    public Logic(int iSize)
    {
        super(iSize);
    }
    public float Average()
    {
        int iSum=0;
        float favg=0.0f;
        
       for(int iCnt=0;iCnt<Arr.length;iCnt++)
       {
          iSum=iSum+Arr[iCnt];
       
        }
        favg=iSum/Arr.length;
       return favg ;
    }

}




class Average
{
  public static void main(String arg[])
  {

    int iSize=0;
    float fRet=0.0f;
    
    System.out.println("Enter the size of the array");
    Scanner sc=new Scanner(System.in);
    iSize=sc.nextInt();
    Logic lobj=new Logic(iSize);
    lobj.accept();
    

     fRet=lobj.Average();
    
   
     System.out.println("Average is:"+fRet);
   
  }
}